doc/source/setup-helm.rst

```diff
@@ -53,6 +53,24 @@ It should provide output like
 
 Make sure you have at least version 2.4.1!
 
+Secure Helm
+~~~~~~~~~~~
+
+Ensure that `tiller is secure <https://engineering.bitnami.com/articles/helm-security.html>`_ from access inside the cluster:
+
+   .. code:: bash
+
+      kubectl -n kube-system patch deployment tiller-deploy --patch '
```

Comment:
You can replace this with the single line:

```
kubectl --namespace=kube-system patch deployment tiller-deploy --type='json' --patch='[{"op": "replace", "path": "/spec/template/spec/containers/0/command", "value": ["/tiller", "--listen=localhost:44134"]}]'
```

Reply:
Okay, in my testing, I had to use the "add" operation rather than "replace".